### ISSUE TITLE
border-box?

### ISSUE THREAD

**Issue by marcandre** [noelboss/featherlight#42]
I noticed that in the demo, the `box-sizing` is bootstrap's `box-content`, and it seems to affect the look to change that. Shouldn't we specify the `box-sizing` in our CSS if it's not the default one?

**Comment by smerriman** [comment]
For what it's worth, the very first thing you see when testing this is library is that the images don't fit properly on the screen, with the bottoms cut off. After debugging for a long time, you realise that even the demo site doesn't work if you remove the call to bootstrap. You search GitHib, and find this was discovered 2 years and and nothing has been done about it. You leave this and never use it again.

**Comment by marcandre** [comment]
I guess I could add the `box-sizing` to our css... The thing is, it would be better if we didn't rely on that I think

**Comment by robneu** [comment]
That's what we [ended up doing](https://github.com/wpsitecare/wp-featherlight/blob/develop/css/wp-featherlight.css#L26-L30), FWIW.

**Comment by marcandre** [comment]
OK, I've added the CSS and released 1.4.2

I'm closing this, but fixes to avoid needing `border-box` welcome...

**Comment by smerriman** [comment]
If you try this on your own site you'll realise the fix doesn't work. The box now fits on the screen, but the image has a scrollbar.

**Comment by marcandre** [comment]
Mmm, indeed I see that the image triggers a scrollbar if
- viewed on a narrow screen
- limited by the height of the window

Looks like this issue was present beforehand though.

I believe I've fixed it by bulletproofing the calculations by rounding down wherever needed, although Chrome hides half the closebox in what seems to be a chrome bug. Firefox is ok.

Released as 1.5.0 (not quite semantic, in a way the CSS change should trigger a v2...)
